Modules/Sources/WordPressKit/ReaderFeed.swift

```diff
@@ -60,16 +62,27 @@ public struct ReaderFeed: Decodable {
 
             let iconContainer = try siteContainer.nestedContainer(keyedBy: IconKeys.self, forKey: .icon)
             blavatarURL = try? iconContainer.decode(URL.self, forKey: .img)
+
+            // Fixes CMM-1002: in some cases, the backend fails to embed certain fields
+            // directly in the feed object
+            if feedURL == nil {
```

Comment:
Not very elegant, but matches the rest of the code and gets the job done.

Both `title` and `url` fields are now optional. Hopefully, for each response, there will be a combination of at least some of these fields. The UI displays what's available.